components/alternatehreflangelements/default.htm

```diff
@@ -0,0 +1,3 @@
+{% for loc, url in __SELF__.locales %}
+    <link rel="alternate" hreflang="{{ loc }}" href="/{{ url }}" />
```

Comment:
Does this support October living in a subdirectory? Could we process `url` before passing it to the view to get the fully qualified URL instead here?

Reply:
Never thought about the 'subdirectory' thing. Never used it. 
We could process `url` indeed and prefix it with `env('APP_URL')` or is there a better way to do this? Or does OctoberCMS provide a helper for this?

Reply:
Hmm forgot, by default octobercms does not use .env files. So probably prefix it with `Config('app.url')` then instead?

Reply:
Give `url()` a try, let me know how it works